pkgs/development/python-modules/pytest-lsp/default.nix

```diff
@@ -3,28 +3,28 @@
   fetchPypi,
   buildPythonPackage,
   hatchling,
-  pygls,
+  pygls_2,
   pytestCheckHook,
   pytest-asyncio,
 }:
 
 buildPythonPackage rec {
   pname = "pytest-lsp";
-  version = "0.4.3";
+  version = "1.0.0";
   pyproject = true;
 
   src = fetchPypi {
     inherit version;
     pname = "pytest_lsp";
-    hash = "sha256-ND9r2i+qMg7V/Ld8lCDScDzlZdHRRP6CfjGYp9wpkRw=";
+    hash = "sha256-uoyVstl1o2Akn/pXaOHm9E2H0Q73dUBw07MhECckovE=";
   };
 
   build-system = [
     hatchling
   ];
 
   dependencies = [
-    pygls
+    pygls_2
```

Comment:
`packaging` is also required since https://github.com/swyddfa/lsp-devtools/commit/5a800cbdd8d276ec6bd01d9bc31c7e1ea47f80b2.